lib/internal/Magento/Framework/Serialize/Serializer/JsonHexTag.php

```diff
@@ -0,0 +1,35 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+declare(strict_types=1);
+
+namespace Magento\Framework\Serialize\Serializer;
+
+use Magento\Framework\Serialize\SerializerInterface;
+
+/**
+ * Serialize data to JSON with the JSON_HEX_TAG option enabled
+ * (All < and > are converted to \u003C and \u003E),
+ * unserialize JSON encoded data
+ *
+ * @api
+ * @since 100.2.0
+ */
+class JsonHexTag extends Json implements SerializerInterface
+{
+    /**
+     * {@inheritDoc}
```

Comment:
Please remove the curly brackets

Reply:
done, any idea why they aren't allowed anymore?